terraform/modules/admin-lb/main.tf

```diff
@@ -30,13 +30,37 @@ resource "kubernetes_manifest" "admin-server-cert" {
   }
 }
 
+resource "null_resource" "wait_for_server_cert_creation" {
+  provisioner "local-exec" {
```

Comment:
Given that you're using bash-specific constructs, you may want to set the [interpreter](https://developer.hashicorp.com/terraform/language/resources/provisioners/local-exec#interpreter) explicitly, instead of relying on what Terraform thinks it's best.

Reply:
done